string id
float32 x
float32 y
float32 yaw
float32 yaw_rate
float32 v
float32 a
float32 r
